### ISSUE TITLE
Add automated tests for accessibility issues

### ISSUE THREAD

**Issue by ewinslow** [Elgg/Elgg#5711]
There are some tools out there that can detect basic violations pretty easily. I think it'd be a great sign of commitment to accessibility if we integrated these into our test suite.

https://code.google.com/p/accessibility-developer-tools/wiki/AuditRules

**Comment by jdalsem** [comment]
Would be nice to have this. Can reopen this ticket if someone is willing to take this on.
